source/3-landing-page-cicd/cdk/bin/landing-page.ts

```diff
@@ -9,7 +9,11 @@ const app = new cdk.App();
 // Use for direct deploy to an environment without pipeline
 new LandingPageStage(app, 'Test', {});
 // Use to deploy the pipeline stack
-const pipelineStack = new LandingPagePipelineStack(app, 'LandingPageStackPipeline');
+const pipelineStack = new LandingPagePipelineStack(app, 'LandingPageStackPipeline', {
+    env: {
```

Comment:
This is to ensure the `region` value is available at the synth time. Since we deploy the stack using a particular `--profile cicd`, we can obtain the region from the AWS profile.